lib/utils/packaging/unarchive.go

```diff
@@ -129,9 +129,6 @@ func replaceZip(toolsDir string, archivePath string, extractDir string, execName
 				return trace.Wrap(err)
 			}
 			appPath := filepath.Join(toolsDir, baseName)
-			if err := os.Remove(appPath); err != nil && !os.IsNotExist(err) {
-				return trace.Wrap(err)
-			}
 			// For the Windows build we have to copy binary to be able
 			// to do this without administrative access as it required
 			// for symlinks.
```

Comment:
If I understand correctly, `utils.CopyFile` already overwrote the destination file? This is why removing it before is not needed?

Reply:
yes, it was not necessary, and windows return error if we try to self-remove

Reply:
It might be good to leave a comment about it, someone touching this code in 6 months is likely not going to remember about this Windows-specific behavior.